### DISCUSSION TITLE
NAutoComplete - How to disable automatic selection of first option?

### DISCUSSION THREAD

**Discussion by linkurzweg** [tusen-ai/naive-ui#4661] [Q&A]
Hello,

I want to create a combined autocomplete/search field with NaiveUI. I'm using NAutoComplete and it works fine basically, but there is one problem: 

When I enter a text into the autocomplete input field and press Enter afterwards, the first autocomplete options is selected automatically. I don't want this behavior. Pressing Enter should trigger a search instead. Autocomplete options should only be selected manually by clicking them with the mouse or using the arrow keys.

Is there any way to achieve this kind of behavior?

Thanks!

**Comment by distor-sil3nt** [comment]
Hello @07akioni and naive-Community,

is there any way to alter the default behavior of the keypress event (enter / submit) for the autocomplete component? We would like to submit the actual search input / entry of the user on keypress enter. Not the first autocomplete option which is currently taken when enter is hit.
If there is a setting to change the default (first autocomplete option is taken) to custom (submit the actual inputted search term), please let us know.

Cheers!
